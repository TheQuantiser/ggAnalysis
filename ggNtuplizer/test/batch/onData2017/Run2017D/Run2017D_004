file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/04985930-5E37-E811-A6C2-1866DAEA8808.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/5043846C-4637-E811-8ADA-1866DAEB3628.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/8232BBD3-7237-E811-82FD-1866DAEA6C40.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/9ED94565-4837-E811-984A-B083FED42FAF.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/3EE08C94-3A37-E811-9E77-B083FED045EC.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/5295DBB1-6937-E811-B1EE-1418774120C3.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/0AB09454-4037-E811-8FF5-549F3525DDFC.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/A8A2B86F-3E37-E811-BD5D-1866DAEA79C8.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/E49E527C-6E37-E811-A1AB-001C23BED7E1.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/E487323F-4237-E811-8CEF-1866DAEB1FC8.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/A8493D01-7C37-E811-ABF3-1866DAEB5C94.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/9CA60B0F-6337-E811-ADB2-801844DEF494.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/D0109DBA-6937-E811-97F1-141877411EA2.root
