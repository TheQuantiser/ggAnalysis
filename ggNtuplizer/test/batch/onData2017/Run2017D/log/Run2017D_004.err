condor_exec.exe: line 5: /cvmfs/cms.cern.ch/cmsset_default.sh: No such file or directory
condor_exec.exe: line 9: scramv1: command not found
condor_exec.exe: line 11: cmsRun: command not found
