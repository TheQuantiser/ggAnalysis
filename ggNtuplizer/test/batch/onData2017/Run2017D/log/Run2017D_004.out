scorpion3.spa.umn.edu
